docs/12-self-hosting/03-host-creation/03-QEMU/04-windows.mdx

```diff
@@ -0,0 +1,456 @@
+import Vnc from '/assets/images/vnc-connection.png';
+import Virtio from '/assets/images/win10-virtio-drivers.png';
+import BootFromCd from '/assets/images/boot-from-cd.png';
+import BootFromCd2 from '/assets/images/boot-from-cd2.png';
```

Comment:
Running this branch locally, I get the following runtime error:

Uncaught runtime errors:
×
ERROR
Cannot find module '/assets/images/boot-from-cd2.png'
    at webpackMissingModule (webpack-internal:///./docs/12-self-hosting/03-host-creation/03-QEMU/04-windows.mdx:16:50)
    at eval (webpack-internal:///./docs/12-self-hosting/03-host-creation/03-QEMU/04-windows.mdx:16:157)
    at ./docs/12-self-hosting/03-host-creation/03-QEMU/04-windows.mdx (http://localhost:3000/content---docs-self-hosting-host-creation-qemu-windowsc-4-d-e17.js:22:1)
    at __webpack_require__ (http://localhost:3000/runtime~main.js:36:33)
    at Function.fn (http://localhost:3000/runtime~main.js:347:21)

Reply:
added the missing file in https://github.com/game-ci/documentation/pull/419/commits/2be1df01fc771def46de638bac5fcc47a1fc0992